allure-generator/src/main/javascript/components/widgets-grid/WidgetsGridView.js

```diff
@@ -71,7 +71,8 @@ class WidgetsGridView extends View {
 
         this.addRegion(name, {el: el.find('.widget__body')});
        /// fetchAndShow(this, name, this.model, new Widget({model: this.model}))
-        this.getRegion(name).show(new Widget({model: this.model}));
+        const model = this.model;
+        this.getRegion(name).show(new Widget({model, name}));
```

Comment:
You mean this: https://github.com/allure-framework/allure2/blob/master/allure-generator/src/main/javascript/plugins/widget-status/StatusWidgetView.js
So, what is the problem?